src/Text/Libyaml.hs

```diff
@@ -62,12 +63,14 @@ data Event =
     | EventDocumentEnd
     | EventAlias !AnchorName
     | EventScalar !ByteString !Tag !Style !Anchor
-    | EventSequenceStart !Anchor
+    | EventSequenceStart !Tag !SequenceStyle !Anchor
     | EventSequenceEnd
-    | EventMappingStart !Anchor
+    | EventMappingStart !Tag !MappingStyle !Anchor
     | EventMappingEnd
     deriving (Show, Eq)
 
+-- | Style for scalars - e.g. quoted / folded
```

Comment:
Thank you for cleanups like this 👍

Reply:
Well, thank *you* for the whole library. 👍